src/main/java/net/syscon/elite/service/OffenderAddressService.java

```diff
@@ -60,7 +61,7 @@
                                             .addressId(address.getAddressId())
                                             .activeFlag("Y".equalsIgnoreCase(addressUsage.getActiveFlag()))
                                             .addressUsage(addressUsage.getAddressUsage())
-                                            .addressUsageDescription(addressUsage.getAddressUsageType().getDescription())
+                                            .addressUsageDescription(addressUsage.getAddressUsageType() == null ? null : addressUsage.getAddressUsageType().getDescription())
```

Comment:
I know Lombok helps to reduce boiler plate but I think it would really pay to use the Optional feature in Java more on some of our JPA mappings.  Of course this would mean adding a getter to override the Lombok generated one but worth I think.